pkg/services/serviceaccounts/database/database.go

```diff
@@ -28,19 +28,24 @@ func NewServiceAccountsStore(store *sqlstore.SQLStore) *ServiceAccountsStoreImpl
 	}
 }
 
-func (s *ServiceAccountsStoreImpl) CreateServiceAccount(ctx context.Context, sa *serviceaccounts.CreateServiceAccountForm) (saDTO *serviceaccounts.ServiceAccountDTO, err error) {
-	// create a new service account - "user" with empty permissions
-	generatedLogin := "Service-Account-" + uuid.New().String()
+func (s *ServiceAccountsStoreImpl) CreateServiceAccount(ctx context.Context, orgID int64, name string) (saDTO *serviceaccounts.ServiceAccountDTO, err error) {
+	generatedLogin := "sa-" + strings.ToLower(name)
+	generatedLogin = strings.ReplaceAll(generatedLogin, " ", "-")
```

Comment:
question: firstly, we seem to be changing SA login naming scheme from `Service-Account-` + name to `sa-` + name. Does this require any doc updates?

Secondly, what's the reasoning for prefixing login with `sa-`? Is it to avoid name clashes with user logins?

Reply:
- Docs are still not written yet so it should be problematic

- We prefix to avoid clashing with user logins (also more indicative in places where we'll mix both

- we reduced from `Service-Account-` to `sa-` because the IDs were considered too long for good UX